pkg/operation/common/types.go

```diff
@@ -35,6 +35,11 @@ const (
 	// CloudPurposeSeed is a constant used while instantiating a cloud botanist for the Seed cluster.
 	CloudPurposeSeed = "seed"
 
+	// ClusterExpirationTime is an annotation on a Shoot resource whose value represent the time when the Shoot lifetime
+	// is expired. The lifetime can be extended, but at most by the minimal value of the 'clusterLifetimeDays' property
+	// of referenced quotas.
+	ClusterExpirationTime = "expiration.garden.sapcloud.io/expirationTimestamp"
```

Comment:
Hm, doubled `expiration`. Can we name that `shoot.garden.sapcloud.io/expirationTimestamp` (similar to [this](https://github.com/gardener/gardener/blob/master/pkg/operation/common/types.go#L155))?
And the constant should then have the name `ShootExpirationTimestamp` (by naming conventions for these labels/annotations).

Reply:
renamed it